packages/gatsby/src/utils/show-experiment-notice.ts

```diff
@@ -35,3 +56,31 @@ export function showExperimentNoticeAfterTimeout(
     clearTimeout(noticeTimeout)
   }
 }
+
+export const createNoticeMessage = (notices): string => {
+  let message = `\nHi from the Gatsby maintainers! Based on what we see in your site, these coming
+features may help you. All of these can be enabled within gatsby-config.js via
+flags (samples below)`
+
+  notices.forEach(
+    notice =>
+      (message += `
+
+${chalk.bgBlue.bold(
+  terminalLink(notice.experimentIdentifier, notice.umbrellaLink)
+)}, ${notice.noticeText}\n`)
+  )
+
+  return message
+}
+
+const showNotices = (): void => {
+  emitter.off(`COMPILATION_DONE`, showNotices)
+  if (noticesToShow.length > 0) {
+    telemetry.trackFeatureIsUsed(`InviteToTryExperiment`)
```

Comment:
I think this should be an event and not feature? The features get appended to all future events and this makes it quite weird to work with them

Reply:
I just mean - the `trackFeatureIsUsed` method on telemetry is probably not right method for this - as this is not feature that change current mode (+ not something that user can opt-in / opt-out)

Reply:
And also - we likely would want to capture which notices are shown? to be able to see if those notices are even working and maybe consider not showing them / spamming terminal if the rate of inviting to using them is low?

Reply:
yeah events make sense — this is `trackCLI`?

Reply:
yeah, not most intuitive but `trackCLI` create event with `CLI_COMMAND_` prefix as type. But we do use it for example in https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/redux/actions/public.js#L666 which is not CLI related really ...

Maybe we should ask @jamo about this :P

Reply:
yeah `trackEvent` or something

Reply:
yea, `trackCli('SHOW_INVITE_EXPERMENT_EVENT`...` or such is better for actions.